x/ibc/23-commitment/types/merkle_test.go

```diff
@@ -11,6 +11,8 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
+var specs = []string{"ics23:iavl", "simple:v"}
```

Comment:
should we use the string `ProofSpec`s from `ics23` instead to prevent deprecation in case they change?